_layouts/facilitation.html

```diff
@@ -0,0 +1,29 @@
+---
+layout: default
+---
+
+<div class="bg-dark py-2 d-none d-md-block">
+    <div class="container-xxl">
+        <div class="row justify-content-between align-items-center">
+            <div class="col-auto position-absolute">
+                <h5 class="text-secondary mb-0"><i class="bi bi-person-check-fill"></i> Facilitation</h5>
+            </div>
+            <div class="col d-flex">
+                <div class="ms-auto me-md-auto">
+                    <a class="text-secondary" href="/services/facilitation">
+                        <i class="bi bi-house-door-fill"></i> The Team
+                    </a>
+                    <a class="text-secondary" href="https://portal.osg-htc.org/documentation/">
```

Comment:
```suggestion
                    <a class="text-secondary mx-3" href="https://portal.osg-htc.org/documentation/">
```

The links are a bit squished together, maybe add some spacing here